toolkit/scss/shared_placeholders/_dm-typography.scss

```diff
@@ -1,10 +1,30 @@
 @import "_typography.scss";
+@import "_measurements.scss";
 
-%heading-xmedium {
+// All heading styles, except heading-xmedium, are copied from GOVUK Elements as of this commit:
```

Comment:
I've made the comments specific to the `%heading-small` block and moved it to be above it to make it more obvious. Does that help?